src/common/packager.ts

```diff
@@ -3,7 +3,9 @@
 
 import {ChildProcess} from "child_process";
 import {CommandExecutor} from "./commandExecutor";
-import {Log, LogLevel} from "./log";
+import {ErrorHelper} from "./error/errorHelper";
+import {Log} from "./log/log";
+import {LogLevel} from "./log/logHelper";
 import {Node} from "./node/node";
 import {OutputChannel} from "vscode";
```

Comment:
Same here. I searched the code for "vscode" usages in the common folder and the 3 classes I found are the Packager, EntryPointHandler and CommandExecutor.